node-playwright-webkit/Dockerfile

```diff
@@ -1,5 +1,5 @@
-FROM ubuntu:jammy
-ARG NODE_VERSION=16
+FROM --platform=linux/amd64 ubuntu:jammy
```

Comment:
btw do you remember why we use ubuntu here, while we have debian in all other places? even the other playwright images use debian, looks like only webkit and `node-playwright` are using ubuntu

Reply:
node-playwright because we use microsoft's prebuilt images
webkit I honestly don't know, this predates me 😂. Something tells me webkit and debian didn't mix, but maybe they do nowadays 🤷

Reply:
yeah i was expecting all pw images to use ubuntu and was puzzled to find this, lets keep it as is, better not to break more things at once :D